llvm/unittests/TargetParser/TargetParserTest.cpp

```diff
@@ -1996,7 +1996,6 @@ TEST(TargetParserTest, AArch64ExtensionFeatures) {
       AArch64::AEK_D128,         AArch64::AEK_LSE128,
       AArch64::AEK_SPECRES2,     AArch64::AEK_RASv2,
       AArch64::AEK_ITE,          AArch64::AEK_GCS,
-      AArch64::AEK_FPMR,         AArch64::AEK_FP8,
```

Comment:
Why are the tests for `AEK_FP8` removed but `AEK_FP8` itself is not?

Reply:
Fixed